include/cocaine/api/cluster.hpp

```diff
@@ -1,6 +1,6 @@
 /*
-    Copyright (c) 2011-2014 Andrey Sibiryov <me@kobology.ru>
-    Copyright (c) 2011-2014 Other contributors as noted in the AUTHORS file.
+    Copyright (c) 2011-2015 Andrey Sibiryov <me@kobology.ru>
```

Comment:
It's a diff.

We use licenses with ancient rules, which requires special header for each file - okay. But it's change is a separate logic unit.